README.md

```diff
@@ -1,2 +1,41 @@
-# proposals
-This repository holds all the past and current proposals for the Mixxx Software
+# Proposals
+
+This repository holds all the past and current proposals for the Mixxx Software. It’s the single place for
+reviewing, discovering, and working on the design documents. It is also a record of past decisions and approvals.
+
+## Current Proposals
+
+* The [open PRs with the `proposal` label](https://github.com/mixxxdj/proposals/pulls?q=is%3Aopen+is%3Apr+label%3Aproposal) show all the pending proposals.
```

Comment:
@mixxx/developer could you please create the `proposal` label please?

Reply:
I guess not. Some PRs maybe related to chores on the repo (like this one) or updates on previously approved proposal (e.g task list or status update, owner, etc...)

Reply:
Done, and deleted all others.